internal/tags/expand.go

```diff
@@ -14,3 +18,27 @@ func Expand(tagsMap map[string]interface{}) map[string]*string {
 
 	return output
 }
+
+func PointerTo(tagsMap map[string]interface{}) *map[string]string {
+	output := make(map[string]string)
+
+	for i, v := range tagsMap {
+		value, _ := TagValueToString(v)
+		output[i] = value
+	}
+
+	return pointer.To(output)
+}
+
+func ExpandFromPointer(tagsMap *map[string]string) map[string]*string {
```

Comment:
(as above) we should be able to remove these two functions, in favour of using the `tags` package in `go-azure-helpers: https://github.com/hashicorp/go-azure-helpers/tree/main/resourcemanager/tags

Reply:
Yep, sorry missed that one... there are a lot of moving parts with this one... Fixed.